packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -2490,7 +2490,7 @@ type MessagesType = {
   'scenes.security.twostepverification.mobile.title': 'Two-Step Verification - Mobile Phone Number'
   'scenes.security.twostepverification.sms.entercode': 'Enter your verification code below and click submit.'
   'scenes.security.twostepverification.sms.entermobile': 'Enter your mobile number and click Get Code. A verification code will be sent.'
-  'scenes.security.twostepverification.yubi.step1': "1. Insert the Yubikey into your computer's USB port."
+  'scenes.security.twostepverification.yubi.step1': "1. Insert the Yubikey into your computer's UBS port."
```

Comment:
why are you renaming to `UBS`? also a few above here changed as well

Reply:
my bad